subsys/bluetooth/audio/csip_crypto.c

```diff
@@ -171,14 +171,8 @@ int bt_csip_sef(const uint8_t k[BT_CSIP_CRYPTO_KEY_SIZE], const uint8_t sirk[BT_
 
 	LOG_DBG("SIRK %s", bt_hex(sirk, BT_CSIP_SIRK_SIZE));
 
-	if (IS_ENABLED(CONFIG_LITTLE_ENDIAN)) {
-		/* Swap because aes_cmac is big endian
-		 * and we are little endian
-		 */
-		sys_memcpy_swap(k1_tmp, k, sizeof(k1_tmp));
-	} else {
-		(void)memcpy(k1_tmp, k, sizeof(k1_tmp));
-	}
+	/* Swap because aes_cmac is big endian and k1 is little endian */
```

Comment:
```suggestion
	/* Swap because aes_cmac is big endian and k is little endian */
```

`k1` is the function, `k` is the little-endian LTK :)

Reply:
k :wink: